### ISSUE TITLE
There are several problems with the postexertional malaise phenotype

### ISSUE THREAD

**Issue by octocat2317** [obophenotype/human-phenotype-ontology#7694]
Dear editors, 

In its current form the phenotype description and classification is misleading and lacks specificity. Let me help you fix this.

The term postexertional malaise (PEM) came from the literature on myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS). The largest review on ME/CFS described PEM as:

> PEM is an exacerbation of some or all of an individual’s ME/CFS symptoms that occurs after physical or cognitive exertion and leads to a reduction in functional ability (Carruthers et al., 2003). As described by patients and supported by research, PEM is more than fatigue following a stressor. Patients may describe it as a post-exertional “crash,” “exhaustion,” “flare-up,” “collapse,” “debility,” or “setback.”2 PEM exacerbates a patient’s baseline symptoms and, in addition to fatigue and functional impairment, may result in flu-like symptoms (e.g., sore throat, tender lymph nodes, feverishness); pain (e.g., headaches, generalized muscle/joint aches) ; cognitive dysfunction (e.g., difficulty with comprehension, impaired short-term memory, prolonged processing time) nausea/gastrointestinal discomfort; weakness/instability; lightheadedness/vertigo; sensory changes (e.g., tingling skin, increased sensitivity to noise); depression/anxiety; sleep disturbances (e.g., trouble falling or staying asleep, hypersomnia, unrefreshing sleep) ; and difficulty recovering capacity after physical exertion. In some cases, patients experience new symptoms as part of the PEM response.

National Academies of Sciences, Engineering, and Medicine. 2015. Beyond Myalgic Encephalomyelitis/Chronic Fatigue Syndrome: Redefining an Illness. Washington, DC: The National Academies Press. https://doi.org/10.17226/19012.

PEM tends to be delayed with respect to the trigger, by hours or even days. This is a key feature that distinguishes it from other types of exertion intolerance where the symptoms tend to occur during the exertion or shortly after it.

The trigger for PEM can also be activities that are not exercise, such as mental activity or sensory stimulation.

I suggest the following changes

1. Postexertional malaise be classified as a type of exertion intolerance, not as type of fatigue. Or as type of abnormal response to exertion.

2. References to "postexertional fatigue" and "exercise-induced fatigue" be removed to avoid confusion. While fatigue is part of PEM and a symptom of ME/CFS, it is very inaccurate to treat "postexertional fatigue" and "exercise-induced fatigue" as synonyms of PEM.

3. The description text be changed to "Postexertional malaise is an exacerbation of some or all of an individual’s ME/CFS symptoms that occurs after physical or cognitive exertion and leads to a reduction in functional ability. Postexertional malaise often occurs with a delay of hours or days."

4. The current gene and disease assocations are I suspect all false associations resulting from an inaccurate definition of PEM as a type exercise related fatigue. They need to checked and removed if inaccurate. I have not looked in depth at all of them. The glycogen storage disease and myofibrillar myopathy association look like false associations to me.  In my opinion an association between PEM and a disease should only be reported if the publication mentions PEM as symptom and it is clear that they're defining PEM in a similar manner as the National Academies report (misunderstanding PEM as fatigue after exercise is a common error).

5. The following disease associations associations should be added: ME/CFS, post-acute sequelae of SARS-CoV-2 infection (PASC). PEM has also been reported to occur in some patients with mitochondrial disorders, for example here https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4748504/. It has also been reported in patients with cancer-related fatigue https://pubmed.ncbi.nlm.nih.gov/32105793/

6. That the aforemtioned National Academies report be added to the relevant publications, as well as this article by the NIH https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530890/

Thank you.

**Comment by octocat2317** [comment]
PS: with point 1. I'm referring to the "Neighbors" button that when clicked appears show postexertional malaise as subtype of fatigue.

**Comment by pnrobinson** [comment]
@octocat2317  -- I agree with your assessment, thanks for these suggestions. Instead of a new term exercise-related fatigue, I think that the other diseases can probably just be annotated with Exercise intolerance HP:0003546.

**Comment by octocat2317** [comment]
Also, I made a copy and paste error at point 3. The phenotype description should not say "an individual’s ME/CFS symptoms" but "an individual’s symptoms" to allow the possibility of PEM being present in other diseases.

**Comment by pnrobinson** [comment]
@octocat2317 -- the definition above is a description of multiple symptoms rather than just postexertional malaise. The HPO attempts to break things down into component parts, and is an abstract classification of phenotypic abnormalities (symptoms, signs, etc) rather than of diseases. Therefore, I am revising the definition above to remove reference to 'joint aches' etc (which of course are part of the clinical picture of ME/CFS, but they would be coded with [Arthralgia](https://hpo.jax.org/app/browse/term/HP:0002829) when using HPO.

Please assess the following proposed definition

Def:  Postexertional malaise (PEM) refers to malaise (a feeling of weakness, overall discomfort, illness, or simply not feeling well) that occurs following physical or cognitive exertion and leads to a reduction in functional ability or exacerbation of other medical problems. 
Comment: PEM can be observed in individuals with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), as a post-acute sequela of SARS-CoV-2 infection (long COVID), and occasionally in individuals with mitochondrial disorders or cancer related fatigue.
PMID:32105793
PMID:33071931
PMID:26904705
Note that our system requires pubmed IDs to link to, and so we cannot automatically link to the NAS article.

Additionally, I have moved the term to be a child of Exercise intolerance and moved that to be a child of Constitutional symptom, since it is not always a [Functional motor deficit HP:0004302](https://hpo.jax.org/app/browse/term/HP:0004302)

Thanks for your input. We more than welcome feedback like this and are working on adapting HPO analysis to the study of long COVID (https://pubmed.ncbi.nlm.nih.gov/34839263/ and https://www.medrxiv.org/content/10.1101/2022.05.24.22275398v1). Please get in touch if this is of interest peter robinson at jax org.

**Comment by octocat2317** [comment]
The problem I have with emphasizing malaise or fatigue or any symptom in the definition is that PEM isn't one symptom plus a bit of other stuff. PEM is when multiple symptoms get worse together after exertion, _in a recognizable and typical pattern_. It's not the individual symptoms that are important but the pattern.

A typical description of PEM occurring in a patient that recently had an infection could be: "I've started feeling better last week and rode my bike for half an hour. The next day I could barely get out of bed and felt weak. It was as if the infection had come back. It took me a few days before feeling okay again."

What makes this a typical PEM pattern: the patient felt better and ready to increase their activity level. The patient doesn't mention unusual problems during the activity. It is the reaction that occurs on the next day that is problematic. The notices the connection to activity and didn't predict this would happen (their body has not reacted like this before. The symptoms are an atypical response to exercise). The delay before the appearance of symptoms, the prolonged recovery, the drop in function, and the symptoms mentioned by the patient are all consistent with PEM.

If someone asked for more detail, the patient would probably be able to describe more symptoms.

What we're trying to define is a distinct pattern of symptoms getting worse in response to exertion. The pattern is more important than the exact symptoms. Each patient will have their own set of symptoms associated with PEM (with significant overlap between patients). The people who wrote the definition of PEM in the National Academies report understood all this, and that is why they listed associated symptoms last. We don't need to list all symptoms. Mentioning a few core symptoms sounds like a good idea.

How about this definition?

> 
> Postexertional malaise is a disproportionate worsening of symptoms or appearance of symptoms that occurs after physical or cognitive exertion and leads to a reduction in functional ability. Postexertional malaise is often delayed by hours or days. Recovery from postexertional malaise is slow and can take days or weeks or longer. Postexertional malaise can worsen a wide range of symptoms, but the more common symptoms include exhaustion, cognitive impairment, muscle pain, muscle weakness, malaise. 
> 
> Comment: PEM can be observed in individuals with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), as a post-acute sequela of SARS-CoV-2 infection (long COVID), and occasionally in individuals with mitochondrial disorders or cancer related fatigue.

Moving PEM to exercise intolerance is a clear improvement and good enough. But since PEM can also be triggered by cognitive exertion it would be ideal if it was included in that category as well, if it exists. In the ME/CFS world we use the term "exertion intolerance" because we think of it was including both physical and cognitive exertion. That PEM is triggered by both is probably an important clue to the underlying defect.

English is a second language for me and some of my sentences might be worded poorly. Don't hesitate to improve them.

I'll look at the long covid stuff tomorrow.

**Comment by octocat2317** [comment]
The NAS report PMID:25695122

**Comment by pnrobinson** [comment]
@octocat2317  -- The issue with the definition is that if we include "disproportionate worsening of symptoms or appearance of symptoms that occurs after physical or cognitive exertion " as part of the core definition, then the definition becomes very broad and could include a different variety of symptoms for each person. Computationally, this would lead to noise, and it would be better to use additional HPO codes to specificity which symptoms get worse in addition to malaise. 
What do you think of this revision?
Def: Postexertional malaise (PEM) refers to malaise (a feeling of weakness, overall discomfort, illness, or simply not feeling well) that occurs following physical or cognitive exertion and leads to a reduction in functional ability or exacerbation of other medical problems.
Comment: PEM can be observed in individuals with myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS), as a post-acute sequela of SARS-CoV-2 infection (long COVID), and occasionally in individuals with mitochondrial disorders or cancer related fatigue. It is often accompanied by exacerabtion or new onset of exhaustion, cognitive impairment, muscle pain, muscle weakness, or other manifestations. The onset of PEM may occur hours or even days after exertion, and recovery from PEM is often slow, taking place over days or weeks or longer.
PMID:32105793
PMID:33071931
PMID:26904705
PMID:25695122

**Comment by octocat2317** [comment]
If PEM is defined as "worsening of symptoms after exertion" it lacks specificity. We seem to agree on this. But I think this must be solved by adding additional criteria for a worsening of symptoms to count as PEM. This is how the ME/CFS research community has attempted to solve the problem (such delay of symptoms, duration, symptoms being atypical for the person, and so on). It's not perfect but the best that can be done with current knowledge. If you create your own definition of PEM that doesn't closely resemble what experts have come up with, then the result could be a misleading definition of PEM that causes confusion. That might be worse than not having PEM defined at all.

If this anonymous patient cannot convince you, perhaps ask an expert such as Dr Lily Chu, Dr Lucinda Bateman, Dr Peter Rowe.

**Comment by pnrobinson** [comment]
@octocat2317 I am aware of the definition in the context of ME/CFS, but my issue is that (1) the definition of PEM is controversial even within the ME/CFS community (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4295644/), and (2) PEM is not limited to ME/CFS, and the distribution of symptoms with long COVID remains to be well described, and (3) the HPO works computational by breaking things down into the small possible component parts and then annotating diseases or individual affects to capture the full spectrum of disease. This is a difficult term to define, and I tried to put the lowest common denominator in the definition and to provide additional context in the comments. Here is an example of how one would define a patient course (https://onlinelibrary.wiley.com/doi/full/10.1002/ggn2.202200016).

**Comment by octocat2317** [comment]
Re. point 1, in my opinion Jason et al's view of PEM was too unspecific. Their view is reflected in the PEM subscale of the DePaul questionnaire that they developed.

A few years ago the NIH worked on Common Data Elements for ME/CFS and there was some discussion in the patient community about what to use to measure PEM. [In a poll organized by the Science for ME forum, a sample of 750 patients clearly preferred the National Academies definition. 
](https://www.s4me.info/attachments/pem-cde-submission-from-s4me-pdf.1640/) 

I have attempted to give a definition of PEM that aligns with that given by the National Academies. 

I promise to look at point 3 tomorrow.

**Comment by octocat2317** [comment]
I would also like to point out that the WHO recently published guidelines on the clinical management of covid-19 and they defined of postexertional malaise like this:

> Post-exertional symptom exacerbation (PESE), also referred to as post-exertional malaise (PEM), is defined as the worsening of symptoms that can follow minimal cognitive, physical, emotional, or social activity, or activity that could previously be tolerated [340]. Symptoms typically worsen 12 to 72 hours after activity and can last for days or even weeks, sometimes leading to a relapse [340][341][342][343]. PESE can contribute to the episodic nature of disability in post COVID-19 condition, often presenting as unpredictable fluctuations in symptoms and function [344][345].

https://app.magicapp.org/#/guideline/j1WBYn/section/jOpQqB

I suspect that if you asked the authors of this guideline why they used the term PESE, they would say because it more accurately describes the phenomenon, whereas PEM can be easily misunderstood as a single symptom (malaise) occurring after exertion. PEM is however the term that has become established in the literature.

**Comment by pnrobinson** [comment]
@octocat2317 
I am wondering if it would be better to use the definition you propose but to choose the term label (primary name) of
Postexertional symptom exacerbation
, and keep postexertional malaise as a synonym. There are a lot of misguided names in the field (arguably, the name Chronic Fatigue Syndrome is partially responsible for the fact that funding for this disease has been much lower than for other diseases with similar prevalence in the population). My problem with the name PEM is that symptom exacerbation is not a specific type of malaise (according to the meaning of the word in the English langauge). In the Human Phenotype Ontology, the terms are not specific for any given disease and we try to make the labels such that they are comprehensible by non-specialists. Here is an example of the usage of Postexertional symptom exacerbation
https://longcovid.physio/post-exertional-malaise

**Comment by octocat2317** [comment]
>I am wondering if it would be better to use the definition you propose but to choose the term label (primary name) of
Postexertional symptom exacerbation, and keep postexertional malaise as a synonym. 

That's a good idea.

**Comment by pnrobinson** [comment]
@octocat2317 thanks so much. I have made the change, which should get onto our homepage with the next release in Oct.
